---
title: 'Representation learning for the identification of bacterial non-coding RNAs'
date: '2021-09-01'
slug: project-representation-learning-for-the-identification-of-bacterial-non-coding-rnas
categories:
  - Open Fall 2021 
  - Open Flexible Timeline
tags:
  - Fall 2021
thumbnailImagePosition: left
thumbnailImage: /Users/ipekensari/Documents/GitHub/site-source/static/img/construction.png
---
Computational methods for identifying protein coding genes can leverage the conserved translational mapping of triplet codons to amino acids. However, non-coding genes, that are transcribed into RNAs but do not code for proteins, lack this structure, hindering their identification. Deep neural networks have shown tremendous promise in learning useful representations of unstructured data, including genomic data. Our lab is investigating the application of deep learning and natural language processing to the learning of representations useful for non-coding gene identification in bacterial genomes. We are seeking a student to contribute to this work. The goals of this project include 1) the identification and application of neural network architectures useful for identifying different classes of non-coding RNAs, 2) the interrogation of well-performing models in order to identify features of non-coding RNAs, and 3) the design of robust test cases which enable the comparison of these novel methods to existing methods for non-coding RNA identification in bacterial genomes.

<!--more-->

{{< alert success >}}
Selected candidate(s) can receive a stipend directly from the faculty advisor. This is not a guarantee of payment, and the total amount is subject to available funding.
{{< /alert >}}

## Faculty Advisor
+ Professor: [Tal Korem](https://koremlab.science)
+ Center/Lab: Korem Lab
+ Location: Presbyterian Hospital 18-200
+ Our research develops data analysis methods for multi-omic microbiome data. We focus on integrating clinical, microbiome, lifestyle and environmental data in a way that advances from statistical associations to actionable insights that can be used in clinical practice.

## Project Timeline
+ Earliest starting date: 9/7/21
+ End date: 5/1/21
+ Number of hours per week of research expected during Fall 2021: ~12

## Candidate requirements
+ Skill sets: 
  Required: Knowledge of python or another programming language
  Ideal: Knowledge of ML, experience with linux
+ Student eligibility: ~~freshman~~, ~~sophomore~~, junior, senior, master's
+ International students on F1 or J1 visa: **eligible**
+ Academic Credit Possible: Yes

